internal/app/machined/internal/sequencer/v1alpha1/v1alpha1_sequencer_test.go

```diff
@@ -0,0 +1,116 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+//nolint: scopelint
+package v1alpha1
+
+import (
+	"errors"
+	"sync"
+	"testing"
+	"time"
+
+	machineapi "github.com/talos-systems/talos/api/machine"
+)
+
+type MockSequencer struct {
+	Sequencer
```

Comment:
Embed the default sequencer so we can override the methods and still be able to test the `run` func.